python/src/dataframe.rs

```diff
@@ -93,6 +93,18 @@ impl DataFrame {
         })
     }
 
+    /// Sort by specified sorting expressions
+    fn sort(&self, exprs: Vec<expression::Expression>) -> PyResult<Self> {
+        let exprs = exprs.into_iter().map(|e| e.expr);
+        let builder = LogicalPlanBuilder::from(&self.plan);
```

Comment:
@jorgecarleitao would it make sense to wrap the `DataFrameImpl` instead?

Reply:
I agree that it is not very clean to use the builder; I tried using the `DataFrameImpl` at some point but was unable because the `DataFrameImpl` contains an `ExecutionContext`, and the Python bindings has its own ExecutionContext wrapper.

Reply:
I think we could share `ExecutionContext.state` between the python binding and `DataFrameImpl` at some point, though it is not urgent.